server/app.js

```diff
@@ -88,13 +89,15 @@ const mongodbPromise = MongoClient.connect(mongoURI, {
   ssl: false,
   useNewUrlParser: true,
 }).then(
-  function (res) {
+  async function (res) {
     app.locals.connection = res
     mongodb = res.db()
     app.locals.appDb = res.db()
     app.locals.dataDb = res.db('dpdata')
     res.db().collection('sessions').drop()
 
+    await UserModel.createFirstAdmin(app.locals.appDb, app.locals.dataDb)
```

Comment:
```suggestion
    await UserModel.createFirstAdmin(app.locals.appDb)
```

Reply:
I think that diff is out of date, fixed on latest now anyway